portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -0,0 +1,24 @@
+const get_comments = async () => {
+    const payload = await fetch('/comments');
+    const comments = await payload.json();
```

Comment:
nit: what happens if the server returns an error or the payload is not formatted as JSON?